packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -59,6 +59,41 @@ def test_if_marathon_app_can_be_deployed_with_mesos_containerizer(cluster):
     cluster.marathon.deploy_test_app_and_check(app, test_uuid)
 
 
+def test_if_marathon_pods_can_be_deployed_with_mesos_containerizer(cluster):
+    """Marathon pods deployment integration test using the Mesos Containerizer
+
+    This test verifies that a Marathon pods can be deployed.
+    """
+
+    test_uuid = uuid.uuid4().hex
+
+    pod_definition = {
+        'id': '/integration-test-pods-{}'.format(test_uuid),
+        'scaling': {'kind': 'fixed', 'instances': 1},
+        'environment': {'PING': 'PONG'},
+        'containers': [
+            {
+                'name': 'ct1',
+                'resources': {'cpus': 0.1, 'mem': 32},
+                'image': {'kind': 'DOCKER', 'id': 'busybox'},
```

Comment:
pulling dockers, esp. with mesos containerizer we have found to be very flaky in CI for a mixture of issues. Do we have to use a docker here? Why can't we just use the host binaries / filesystem?

If we have to use a docker, then should use the same image we are using in the one docker pull test we have, `debian:jessie`: https://github.com/dcos/dcos/blob/master/test_util/marathon.py#L61